zipline/algorithm.py

```diff
@@ -1631,13 +1631,6 @@ def get_datetime(self, tz=None):
             dt = dt.astimezone(tz)
         return dt
 
-    def update_dividends(self, dividend_frame):
-        """
-        Set DataFrame used to process dividends.  DataFrame columns should
-        contain at least the entries in zp.DIVIDEND_FIELDS.
-        """
-        self.perf_tracker.update_dividends(dividend_frame)
```

Comment:
We already had removed that method. (This method was referring to a non-existent `update_dividends` in `PerformanceTracker`.